.github/workflows/update-data.yaml

```diff
@@ -0,0 +1,32 @@
+name: Update Contributor CI Data
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 3 * * *'
+
+jobs:
+  extraction:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    - name: Update Data
+
+      uses: vsoch/contributor-ci@0.0.16
+      env:
+        GITHUB_TOKEN: ${{ secrets.CCI_GITHUB_TOKEN }}
+      with: 
+        results_dir: _data/
+        extract: repos repo_metadata topics languages releases stars activity_commits activity_lines
+        extract_save_format: year/month
+
+    - name: Check that results exist
+      run: tree _data
+
+    - name: Push Results
+      run: |
+        git config --global user.name "github-actions"
+        git config --global user.email "github-actions@users.noreply.github.com"
+        git add _data/data/*
+        git commit -m "Automated push with new data results $(date '+%Y-%m-%d')" || exit 0
+        git push origin main || exit 0
```

Comment:
This needs to be master branch - don't let me forget to change it!

Reply:
Another idea is to preserve the current "master" and keep as a legacy or historical branch, and then update the main branch to main (and push this to main) and all automation to main.

Reply:
FWIW, I like the idea of moving to a `main`

Reply:
okay I've updated the PR so the workflows target main, and this means that we will not merge here, but rather when it's ready I'll push to a new main branch, and then we will change the default to main. And we will forever keep the original site in master, for memory! (I've done this a couple of times before and generally like the practice).